yoyodyne/models/expert.py

```diff
@@ -408,82 +431,64 @@ def get_expert(
     train_data: data.Dataset,
     epochs: int = defaults.ORACLE_EM_EPOCHS,
     oracle_factor: int = defaults.ORACLE_FACTOR,
-    sed_params_path: str = None,
+    sed_params_path: str = "",
```

Comment:
My understanding is for cases of typing we want the null values to be the equivalent nulls for the type, no?

Reply:
I'd say yes if this was C. It doesn't matter much in Python and indeed we use `None` like this elsewhere in library because that's how `argparse` works. Here's an example: if you're training from scratch the arg that's a checkpoint to conintue training from will be `None` for instance.

`""` is also used in some circles to mean "read from stdin if this is an input file or write to stdout if this is an output file" and that's not intended here.